src/main/java/seedu/address/ui/CompletableTodoCard.java

```diff
@@ -36,10 +36,12 @@ public CompletableTodoCard(CompletableTodo todo, int displayedIndex) {
     }
 
     /**
-     * Mark this todo as done.
+     * Returns the text of the completed label that is to be displayed.
+     *
+     * @return {@code String} containing the text that is to be displayed.
      */
-    public void markAsDone() {
-        completedLabel.setText(todo.getIsDone() ? "✔" : "");
+    public static String getTextToDisplay(boolean isDone) {
```

Comment:
So for now there is not mechanism to tick the todo off in via the GUI right? It'll just update the GUI accordingly?

Reply:
yes, only can tick off using mark as done.